crates/subspace-networking/src/node_runner.rs

```diff
@@ -249,12 +249,12 @@ impl NodeRunner {
                                 debug!("Random Kademlia query has yielded empty list of peers");
                             }
 
-                            if sender.send(Some(peers)).is_err() {
+                            if sender.send(peers).is_err() {
                                 debug!("GetClosestPeersOk channel was dropped");
                             }
                         }
                         Err(GetClosestPeersError::Timeout { key, peers }) => {
-                            if sender.send(None).is_err() {
+                            if sender.send(Default::default()).is_err() {
```

Comment:
Just a minor thing. Maybe we can use `vec![]` for clarity

Reply:
Ok.